docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -392,6 +392,35 @@ In such cases configuring Quarkus to recognize the original headers forwarded by
 
 `quarkus.oidc.authentication.force-redirect-https-scheme` property may also be used when the Quarkus application is running behind a SSL terminating reverse proxy.
 
+== Cloud Services
+
+=== Google Cloud
+
+You can have Quarkus OIDC `web-app` applications access `Google Cloud services` such as `BigQuery` on behalf of the currently authenticated users who have enabled OpendId Connect (Authorization Code Flow) permissions to such services in their Google Developer Consoles.
+
+It is super easy to do with https://github.com/quarkiverse[Quarkiverse] https://github.com/quarkiverse/quarkiverse-google-cloud-services[Google Cloud Services], only add
```

Comment:
Maybe, instead of linking to the Quarkiverse Github organisation explaing (in a note) what Quarkiverse is with emphasis on the fact that it's "external contributions". @gastaldi I wonder if there is an "official" description of what is the Quarkiverse because on Github its description is "Home of the Quarkus community extensions" but Quarkus is community based, the platform also so it's not very clear how it is different from a core Quarkus extension or one from the platform ...

Reply:
Hi @loicmathieu, @gastaldi here I wanted to highlight that it exists :-), as opposed to trying to explain what it is about, which would likely be out of scope for the oidc web-app docs. By the way my understanding it is not only about the external contributions, CC @aloubyansky.  
Perhaps we should have a dedicated Quarkiverse introduction document describing its purpose, structure, etc, and, once it becomes available, we'd update this doc with a new link.

Does it sound reasonable ?

Reply:
Yes, we should add a note of some sort to the Quarkiverse wiki/docs on how it is different from the platform. I added this general platform doc recently https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/platform.adoc#quarkus-platform
But it still makes sense to highlight the difference between the two in Quarkiverse wiki.

Reply:
FYI here is an attempt to highlight the diff https://github.com/quarkiverse/quarkiverse/pull/17